### ISSUE TITLE
QRCode Stored inconsistently

### ISSUE THREAD

**Issue by Madrobot9182** [CMPUT301F24pekkas/mini-pekkas#126]
I have determined that the Base64 String that is stored in Firebase from EventCreateFragment, and CameraFragment are inconsistent, which causes the camera scanner to not find the right event. What is being scanned and what is being stored is different.
---
### What is initialized and stored into the event (EventCreateFragment from line 78):
```
Bitmap qrCodeBitmap = QRCodeGenerator.generateQRCode(uniqueQrData, 300, 300);
String qrCodeBase64 = bitmapToBase64(qrCodeBitmap);
```
### With bitmapToBase64 on line 176:
```
ByteArrayOutputStream baos = new ByteArrayOutputStream();
bitmap.compress(Bitmap.CompressFormat.PNG, 100, baos);
byte[] byteArray = baos.toByteArray();
return Base64.encodeToString(byteArray, Base64.NO_WRAP);
```

### And what is queried from scanning with camera (CameraFragment line 85):
```
IntentResult result = IntentIntegrator.parseActivityResult(requestCode, resultCode, data);
...
String scannedData = result.getContents();
String base64QRCode = Base64.encodeToString(scannedData.getBytes(), Base64.NO_WRAP);
```

Tldr: qrCodeBase64 != base64QRCode.

**Comment by Madrobot9182** [comment]
Current Update: I'm attempting to change QR code generation from just completely random UUID to document ID

**Comment by danieladeremi** [comment]
no longer different!
